packages/ppf-contracts/contracts/PPF.sol

```diff
@@ -2,9 +2,10 @@ pragma solidity 0.4.24;
 
 import "./IFeed.sol";
 import "./open-zeppelin/ECRecovery.sol";
+import "@aragon/os/contracts/common/TimeHelpers.sol";
 
 
-contract PPF is IFeed {
+contract PPF is IFeed, TimeHelpers {
```

Comment:
An extra `block.timestamp` was added in #17 ([L101](https://github.com/aragon/ppf/blob/310bbd959da2411123281899b4bd2de4eb6078af/packages/ppf-contracts/contracts/PPF.sol#L101)) which we should update as well